controller/raft/member.go

```diff
@@ -1,17 +1,17 @@
 /*
```

Comment:
Can you revert this change? It's the only change to the file and it's inconsistent with the copyright blocks in other files